session/bootstrap.go

```diff
@@ -1714,6 +1732,10 @@ func doDDLWorks(s Session) {
 	mustExecute(s, CreateColumnStatsUsageTable)
 	// Create table_cache_meta table.
 	mustExecute(s, CreateTableCacheMetaTable)
+	// Create stats_history table.
+	mustExecute(s, CreateStatsHistory)
+	// Create stats_meta_history table.
+	mustExecute(s, CreateStatsMetaHistory)
```

Comment:
We should create a new `ver` for them, here is an example https://github.com/pingcap/tidb/pull/26294/files, you can see changes of `bootstrap.go` in this PR.